templates/patterns/featured_media/featured_media.ui_patterns.yml

```diff
@@ -33,8 +33,8 @@ featured_media:
     title:
       type: text
       label: "Heading"
-      description: "The heading, displayed on the top."
-      preview: "Lorem ipsum"
+      description: 'Title of the featured media.'
+      preview: 'This is the title of this featured media.'
```

Comment:
I missed it the first time around but you can add a title tag here and default it to h2

Reply:
done.